source/write-operations/replace-documents.txt

```diff
@@ -0,0 +1,212 @@
+.. _java-rs-write-replace:
+
+=================
+Replace Documents
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code examples, write, replace, change
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to replace
+a document in a MongoDB collection by performing replace operations.
+
+A replace operation replaces one document in a MongoDB collection.
+You can perform a replace operation by using the ``replaceOne()`` method.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. 
+
+To learn how to create a free MongoDB Atlas cluster and load the sample
+datasets, see the :ref:`<java-rs-getting-started>` tutorial.
+
+.. include:: /includes/reactor-note.rst
```

Comment:
S: move this note out of the "sample data" section and instead put it into the overview